toolz/functoolz.py

```diff
@@ -547,6 +548,27 @@ def compose(*funcs):
         return Compose(funcs)
 
 
+def pipeline(*funcs):
+    """ Compose functions to operate in series.
+
+    Returns a function that applies other functions in sequence.
+
+    Functions are applied from left to right so that
+    ``mkpipe(f, g, h)(x, y)`` is the same as ``h(g(f(x, y)))``.
```

Comment:
`mkpipe` should be `pipeline`

Reply:
Thanks for the review. I have addressed it in 44983f7.